src/Features/CSharp/Portable/ConvertCast/CSharpConvertTryCastToDirectCastCodeRefactoringProvider.cs

```diff
@@ -54,6 +54,10 @@ protected override CastExpressionSyntax ConvertExpression(BinaryExpressionSyntax
             var newLeadingTrivia = asExpression.GetLeadingTrivia().AddRange(asExpression.OperatorToken.TrailingTrivia.SkipInitialWhitespace());
             typeNode = typeNode.WithoutTrailingTrivia();
 
+            // Make sure we make type nullable when converting expressions like `null as string` -> `(string?)null`
+            if (expression.IsKind(SyntaxKind.NullLiteralExpression) && nullableContext.HasFlag(NullableContext.AnnotationsEnabled))
+                typeNode = NullableType(typeNode, Token(SyntaxTriviaList.Empty, SyntaxKind.QuestionToken, SyntaxTriviaList.Empty));
```

Comment:
why the empty trivia lists?

Reply:
If you llok at the CI of second commit of the PR you will see, that if I don't explicitly set empty trivia list here, the factory generates a token with whitespace trivia before and after the token, which breaks the test

Reply:
Sounds like a bug in the handling of elastic markers around `?` tokens.

Reply:
As it turned out, this is only test problem. Removing `SyntaxTriviaList.Empty` doesn't break anything when launched via `RoslynDeployment`:
![NfXndd2TZM](https://user-images.githubusercontent.com/70431552/195284507-f1723f83-f8b2-4ec2-a60f-d492f2d3c34e.png)
Talked about this with Cyrus. He is gonna investigate the problem soon